packages/x-components/src/components/base-dropdown.vue

```diff
@@ -322,12 +322,7 @@
       watch(
         highlightedItemIndex,
         highlightedItemIndex => {
-          nextTick(() => {
-            if (itemsButtonRefs && isInRange(highlightedItemIndex, [0, props.items.length - 1])) {
-              const newItem = itemsButtonRefs?.value?.[highlightedItemIndex];
-              newItem?.focus();
-            }
-          });
```

Comment:
`itemsButtonRefs` is always true, as it is a `ref` (object)
`itemsButtonRefs.value` is always true as it is an array `[]`
so, it can be simplified a lot.